databricks/koalas/tests/test_groupby.py

```diff
@@ -852,6 +852,70 @@ def test_rank(self):
             pdf.groupby([("x", "a"), ("x", "b")]).rank().sort_index(),
         )
 
+    def test_cumcount(self):
+        pdf = pd.DataFrame(
+            {"a": [1, 1, 1, 4] * 3, "b": [None, 0.1, 20.0, None] * 3, "c": [4, 3, 2, 1] * 3},
+            index=np.random.rand(4 * 3),
+        )
+        kdf = ks.from_pandas(pdf)
+
+        self.assert_eq(
+            kdf.groupby("b").cumcount().sort_index(), pdf.groupby("b").cumcount().sort_index()
+        )
+        # TODO: Enable the following test when ks.groupby(dropna=True)
+        # is implemented (see #1007)
+        # self.assert_eq(
+        #    kdf.groupby(["a", "b"]).cumcount().sort_index(),
+        #    pdf.groupby(["a", "b"]).cumcount().sort_index(),
+        # )
```

Comment:
Shall we use the same test dataset as the other `cumxxx` tests for now? Then we can enable this and one more below. Let's see #1007 for this case.

Reply:
@tomspur Could you update the tests?

Reply:
I just changed above the columns from `["a", "b"]` to `["a", "c"]`, which does not contain Nones to have some other and keep the comments to remember to enable testing Nones at least partially.

Reply:
Used the same dataset like the other tests for now. Do you want to add a None to them once #1007 is implemented?

Reply:
Yes, we should modify or add tests which include `None`s in groupkeys at #1007.